lib/commands/std-authenticate.ts

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) 2024. SailPoint Technologies, Inc. All rights reserved. */
+
+import { AccountSchema, Attributes, ObjectInput, ObjectOutput, Permission } from './command'
+
+/**
+ * Input object of `std:authenticate` command
+ */
+export type StdAuthenticateInput = ObjectInput & {
+    username: string
+	password: string
+    options? : any
```

Comment:
Do we know what the set of `options` that could be passed here are? It would be nice to have something more specific than `any`

Reply:
ISC Connectors support pass through authentication which may require additional information along with the password. For example, RSA connector has two ways to authenticate the users: using password and passcode. The additional information to distinguish between password and passcode via ‘options’. Hence the type of ‘options’ is Map<String,Object> which enables supplying almost any info needed by the connectors without restrictions. Therefore, the type is kept as ‘any’ to keep inline with the ISC concept.

Reply:
Gotcha. Are `username` and `password` both needed for the passcode use case?

Reply:
Yes

Reply:
We might want to make password optional if there are going to be cases where it is just username and options

Reply:
Password is required for PTA for most of the managed system. It's just some systems require Options as well.